tests/test_apiclient.py

```diff
@@ -1215,6 +1215,18 @@ def test_put_signed_agreement_on_hold(self, data_client, rmock):
         assert rmock.call_count == 1
         assert rmock.last_request.json() == {'updated_by': 'Chris'}
 
+    def test_countersign_agreement(self, data_client, rmock):
+        rmock.post(
+            "http://baseurl/agreements/101/countersign",
+            json={},
```

Comment:
Yeah, I had a think about that. As far as I can tell all testing the result would do would be testing the mock I created. The below is a passing test for example.

```
def test_countersign_agreement(self, data_client, rmock):
        rmock.post(
            "http://baseurl/agreements/101/countersign",
            json={'socks': True},
            status_code=200
        )

        result = data_client.countersign_agreement(101, 'Chris')

        assert result == {'socks': True}
        assert rmock.call_count == 1
        assert rmock.last_request.json() == {'updated_by': 'Chris'}
```

I'm not sure why the other tests are testing that way. What do you think?